backend/middlewares/auth.js

```diff
@@ -0,0 +1,26 @@
+import ErrorHandler from "../utils/error-handler.js";
+import { ErrorMessages } from "../constants/server-errors.js";
+// import { User } from "../models/user.model.js";
```

Comment:
Why are we keeping the commented code? Is there any specific reason?

Reply:
We will be using the user model for authentication and authorization Middleware. Right now we haven't setup one but once created we will be needing this code.

PS: This is just a starter code.